test/common/cdp.py

```diff
@@ -14,9 +14,9 @@
 TEST_DIR = os.path.normpath(os.path.dirname(os.path.realpath(os.path.join(__file__, ".."))))
 
 
-def browser_path(browser):
+def browser_path(browser, show_browser):
     if browser == "chromium":
-        return browser_path_chromium()
+        return browser_path_chromium(show_browser)
     elif browser == "firefox":
         return browser_path_firefox()
```

Comment:
Showing firefox browser should work as well ideally. Is that just an oversight, or not technically possible right now?

Reply:
Firefox works always. This is the part where we look for browser executable. Every firefox executable we can find supports normal mode. Problem is that we also search for headless chromium, which we don't want to use in this case.

Reply:
Of course, thanks!